packages/core/src/Modules/Cashier/Components/Receipt/account-transfer-receipt.jsx

```diff
@@ -7,141 +7,141 @@ import { formatMoney, getCurrencyDisplayCode, getCurrencyName, routes } from '@d
-                    <Localize i18n_default_text='Your funds have been transferred' />
-                </h2>
-                <div className='cashier__transferred-amount cashier__text--bold'>
-                    {formatMoney(selected_from.currency, receipt.amount_transferred, true)}
-                    <span className={classNames('symbols', `symbols--${selected_from.currency?.toLowerCase()}`)}>
-                        {getCurrencyDisplayCode(selected_from.currency)}
-                    </span>
-                </div>
-                <div className='cashier__transferred-details-wrapper'>
-                    <span className='account-transfer__transfer-details-from'>
-                        <div className='cashier__transferred-details'>
-                            <div className='cashier__text--bold cashier__text--right'>
-                                {selected_from.is_mt ? selected_from.text : getCurrencyName(selected_from.text)}
-                            </div>
-                            <div className='cashier__text--faint'>{selected_from.value}</div>
-                        </div>
-                        <Icon
-                            icon={
-                                selected_from.mt_icon
-                                    ? `IcMt5-${selected_from.mt_icon}`
-                                    : `IcCurrency-${selected_from.currency?.toLowerCase()}`
-                            }
-                            size={32}
-                        />
-                    </span>
-                    <Icon className='cashier__transferred-icon' icon='IcArrowLeftBold' />
-                    <span className='account-transfer__transfer-details-to'>
-                        <Icon
-                            icon={
-                                selected_to.mt_icon
-                                    ? `IcMt5-${selected_to.mt_icon}`
-                                    : `IcCurrency-${selected_to.currency?.toLowerCase()}`
-                            }
-                            size={32}
-                        />
-                        <div className='cashier__transferred-details'>
-                            <div className='cashier__text--bold'>
-                                {selected_to.is_mt ? selected_to.text : getCurrencyName(selected_to.text)}
-                            </div>
-                            <div className='cashier__text--faint'>{selected_to.value}</div>
+    return (
+        <div className='cashier__wrapper account-transfer__receipt'>
+            <h2 className='cashier__header'>
+                <Localize i18n_default_text='Your funds have been transferred' />
+            </h2>
+            <div className='cashier__transferred-amount cashier__text--bold'>
+                {formatMoney(selected_from.currency, receipt.amount_transferred, true)}
+                <span className={classNames('symbols', `symbols--${selected_from.currency?.toLowerCase()}`)}>
```

Comment:
Could you check if we have corresponding styles? AFAIK we're removed the `symbols`.

Reply:
Yeah, I can't seem to find anything related to `symbols--`